src/store/reducers/UserSlice.ts

```diff
@@ -66,8 +70,16 @@ export const counterSlice = createSlice({
           : todo
       );
     },
+    checker: (state) => {
+      state.checker = !state.checker;
+    },
+    filterTodos: (state, action: PayloadAction<string>) => {
+      action.payload === "сlearCompleted" && (state.todos = state.todos.filter((todo) => todo.completed === false));
+      (action.payload === "active" || "completed" || "all") && (state.filtValue = action.payload);
```

Comment:
Use enum Condition

Reply:
really, thanks, I forgot about the slice file where enum also used